src/UI/templates/js/libs/jquery.dragster.js

```diff
@@ -0,0 +1,85 @@
+// 1.0.3
```

Comment:
Please move to global bower libs as soon as accepted by JF and available in core, see other PR concerning this. If not accepted, this may stay.